about.html

```diff
@@ -0,0 +1,150 @@
+                <li class="sub-nav-item">My Page</li>
+                <li class="sub-nav-item">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <header class="navbar fixed-top-nav" id="top-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-icon"></i>
+            <div class="desktop-nav">
+                <a href="index.html"><img class="logo-nav" src="./images/aspire-logo.png" width="110px"
+                        alt="Aspire logo"></a>
+                <ul class="nav-list float-right">
+                    <li><a class="nav-list-item" href="index.html">Home</a></li><br>
+                    <li><a class="nav-list-item" href="#about-myself">Program</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">Join</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">Sponsor</a></li><br>
+                    <li><a class="nav-list-item" href="#my-form">News</a></li><br>
+                    <li><a href="#" class="mt-3  p-2" id="btn-nav">Company</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </header>
+
+
+    <main class="main-section text-center">
+        <section class="top-bg">
+            <section class="hero container">
+                <br>
+                <h4 class="greetings">"Explore Your Dream Career with Aspire"</h4>
+                <h2 class="hero-title">Get connected <br /> to the experts Globally</h2>
+                <br>
+            </section>
+
+            <section class="message">
+
+                <div class="container">
+                    <p class="message-p p-3 border border-white">
+                        At Career Guiders, we help young people explore different career options with the help of
+                        industry experts. Our mission is to reduce unemployment in Africa by funding talents and
+                        connecting young people with experts in their field of interest. Join us today and start your
+                        journey to success!
+                    </p>
+                    <p>please contact us per Email for any further questions.</p>
+                    <b class="underline">komezarsene15@gmail.com</b>
+                </div>
+            </section>
+            <br><br><br>
+        </section>
+        <section class="about-program bg-white">
+            <div class="container">
+                <br>
+                <h4 class="text-center section-title-black">Our Mission</h4>
+                <hr class="title-line">
+
+                At Career Guiders, we strive to reduce unemployment in Africa by helping young people discover and
+                pursue their interests and talents.
+                </p>
+                <br>
+                <div class="container"><img src="./images/aspire-logo.png" width="90%" height="100px" alt="Aspire logo">
+                </div>
+                <br>
+            </div>
+        </section>
```

Comment:
- Please kindly adjust the width of the logo so that it fit well on the page
- Also kindly add the outer border 

Your design
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/9873f991-9991-4724-ad2d-6b78f991e74e)

Guideline
![image](https://github.com/Arsenethierry/capstone-project-aspire-web/assets/78251927/20a11e46-fd01-42a4-b08d-6526645ce3f7)